README.md

```diff
@@ -234,7 +234,11 @@ For this demo, `PRIVATE_KEY` environment variable was set to an _EOA_ private ke
 
 ### Sending Ether using a Trezor hardware wallet
 
-[![asciicast](https://asciinema.org/a/346692.svg)](https://asciinema.org/a/346692)
+[![asciicast](https://asciinema.org/a/9BrQKYQRXbysmEL8rw1jDWZhn.svg)](https://asciinema.org/a/9BrQKYQRXbysmEL8rw1jDWZhn)
+
+### Create a remove owner transaction in the Transaction Service
```

Comment:
@Uxio0 I suppose that here you would like to say 'Create and remove...'

Reply:
Yes

Reply:
No, create a "remove owner transaction"